ceph/rados/core_workflows.py

```diff
@@ -4615,3 +4615,60 @@ def get_rados_df(self, pool_name: str = None):
         out = self.run_ceph_command(cmd=_cmd, client_exec=True)
 
         return out["pools"][0] if pool_name else out
+
+    def set_service_managed_type(self, service_type, unmanaged) -> bool:
+        """
+        Method to set the service to either managed or unmanaged
+        The service types are- mon,mgr,osd,rgw, mds
+        Args:
+            unmanaged: True or false, for the service management
+
+        returns:
+            Pass -> True, Fail -> false
+        """
+        cmd_export = f"ceph orch ls {service_type} --export"
+        out = self.run_ceph_command(cmd=cmd_export, client_exec=True)[0]
+        if unmanaged:
+            log.debug(
+                f"Setting the {service_type} service as unmanaged by cephadm. current status : {out}"
+            )
+            out["unmanaged"] = "true"
+        else:
+            log.debug(
+                f"Setting the {service_type} service as unmanaged by cephadm. current status : {out}"
+            )
+            out["unmanaged"] = "false"
```

Comment:
Here instead of explicitly setting it to False, we should remove the "unmanaged" key from the dictionary if it exists

Reply:
This should be fine. Would not make difference with workflow.